client/test-helpers/testing-library/index.js

```diff
@@ -4,7 +4,10 @@ import { applyMiddleware, createStore } from 'redux';
 import thunkMiddleware from 'redux-thunk';
 import initialReducer from 'calypso/state/reducer';
 
-const render = ( ui, { initialState, store, reducers, ...renderOptions } = {} ) => {
+export const renderWithProvider = (
```

Comment:
Shall we name it `renderWithReduxProvider` or are we going to add more providers as we need it to that function?

Reply:
Yep, those were my thoughts as well - we can either rename it or add more providers if needed in the future.